src/pudl/transform/eia860.py

```diff
@@ -1023,12 +993,6 @@ def _core_eia860__emissions_control_equipment(
         1000.0 * emce_df["emission_control_equipment_cost"]
     )
 
-    emce_df = (
-        pudl.metadata.classes.Package.from_resource_ids()
-        .get_resource("core_eia860__scd_emissions_control_equipment")
-        .encode(emce_df)
-    )
-
```

Comment:
Removed the encoding step because it's the last part of the transform, and now happens in the harvesting process for this table.